app/components/curriculum-inventory/report-list-item.hbs

```diff
@@ -51,12 +52,13 @@
     <CurriculumInventory::PublicationStatus @item={{@report}} />
   </td>
   <td class="text-right report-actions" colspan="2" data-test-actions>
-    <FaIcon
+    <span role="button" {{on "click" (fn @edit @report)}}>
```

Comment:
I'd like to see this either be a real button or else a link with the icon. Adding a span that acts like a button is a workaround that is a bit of an anti-pattern to avoid doing it the right way and styling with CSS.